charts/logfire/templates/_helpers.tpl

```diff
@@ -139,6 +139,19 @@ spec:
   value: "20000"
 {{- end -}}
 
+{{- define "logfire.ffIngestShared" -}}
+- name: FF_INGEST_MAX_FUTURE_TIME_DRIFT
+  value: "1h"
+- name: FF_INGEST_MAX_PAST_TIME_DRIFT
+  value: "24h"
+- name: FF_INGEST_EXTERNAL_FILE_SPILL_SIZE_THRESHOLD_BYTES
+  value: "131072" # 128kb
+- name: FF_INGEST_VACUUM_TRIGGER_TRESHOLD
+  value: "1000"
+- name: RUST_BACKTRACE
+  value: "1"
+{{- end -}}
```

Comment:
@cetra3 we really should do that refactor to finish splitting up the ingest buffer / ingest processor so we don't bleed unnecessary complexity like this all over the place